### PR TITLE
[go/hertz] Update hertz version and optimise config

### PR DESCRIPTION
**Pull request by Skyenought**
Since the hertz code was found to be inconsistent with fasthttp and fiber, this commit was made to be consistent with them.

### CODE REVIEW

**Comment by li-jin-gou** [comment]
cc @nbrady-techempower

**Comment by Skyenought** [comment]
@nbrady-techempower you can merge code now

File: frameworks/Go/hertz/hello.go [hunk 1]
```diff
@@ -66,122 +72,208 @@ func parseQueries(c context.Context, ctx *app.RequestContext) int {
...
 // / Test 4: Fortunes
 func fortunes(c context.Context, ctx *app.RequestContext) {
-	rows, err := fortuneStatement.Query()
-	if err != nil {
-		ctx.AbortWithError(500, err)
-		return
-	}
-
+	rows, _ := db.Query(context.Background(), fortuneSelect)
 	fortunes := make(Fortunes, 0, 16)
```
Comment (nbrady-techempower) [inline frameworks/Go/hertz/hello.go:105]: Not sure how this was missed before, but you cannot allocate memory with prior knowledge of how many fortunes there are.

https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes

> v. The list data structure must be a dynamic-size or equivalent and should not be dimensioned using foreknowledge of the row-count of the database table.

Comment (Skyenought) [reply frameworks/Go/hertz/hello.go]: This code mimics the code from fiber
```go
fortunes := make([]templates.Fortune, 0, 16)
```

Comment (Skyenought) [reply frameworks/Go/hertz/hello.go]: @nbrady-techempower 
I've also fixed all the early memory allocation issues in the go framework, so please merge as soon as possible if you can, it's very important to me.

**Pull request merged**